### ISSUE TITLE
Missing css property: -webkit-text-decoration

### ISSUE THREAD

**Issue by MrJohz** [frenic/csstype#72]
According to [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration#Browser_compatibility), `text-decoration` requires no browser prefixes *except* for when used with a shorthand property in Safari browsers, when it needs the `-webkit-` prefix.

From what I can tell, the current types don't allow for the `-webkit-` prefix for `text-decoration`.

**Comment by frenic** [comment]
That's right. The current generator is only "looking" in the `text-decoration` row and not the "Shorthand" row. This needs to be fixed. Thanks!
